pkgs/applications/editors/vscode/extensions/default.nix

```
@@ -3190,6 +3190,21 @@ let
         };
       };
 
+      ms-python.flake8 = buildVscodeMarketplaceExtension {
+        mktplcRef = {
+          name = "flake8";
+          publisher = "ms-python";
+          version = "2023.10.0";
+          hash = "sha256-4Vjw8yJPrxLg0hcoTw8AEBEcmQ9sEUNqFaHLxICks0E=";
+        };
+        meta = with lib; {
```

```suggestion
        meta = {
```
- Please, don't introduce nested with.
  https://github.com/NixOS/nixpkgs/tree/master/pkgs/applications/editors/vscode/extensions#conventions-for-adding-new-extensions

- Changelog is missing.

- Consider adding yourself as maintainer.